mage_ai/frontend/components/PipelineDetail/Runs/Table.tsx

```diff
@@ -617,6 +620,22 @@ function PipelineRunsTable({
                       )
                     }
                   </Text>,
+                  <Text
+                    {...SHARED_DATE_FONT_PROPS}
+                    default
+                    key="row_execution_time"
+                    title={startedAt && completedAt 
```

Comment:
can you add parenthesis around `startedAt && completedAt` to make it a bit clearer the condition for the ternary?

Reply:
and below as well